installers-conda/resources/menuinst_cli.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
```

Comment:
Perhaps this should have our copyright? I mean, I think it should given that it's a file developed by us.